scripts/analyze-bundle/command.mjs

```diff
@@ -0,0 +1,172 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+function formatBundleSize(bundleSize) {
+  if (bundleSize === 0) {
+    return '-';
+  }
+
+  return `${Math.round(bundleSize / 1000)} kb`;
+}
+
+function formatBundleSizeDiff(bundleSizeDiff, initialBundleSize) {
+  if (bundleSizeDiff === 0) {
+    return '-';
+  }
+
+  const sign = bundleSizeDiff > 0 ? '+' : '-';
+  const diffPercentage = Math.round(
+    (Math.abs(bundleSizeDiff) / initialBundleSize) * 100
+  );
+
+  return `${sign}${formatBundleSize(
+    Math.abs(bundleSizeDiff)
+  )} (${sign}${diffPercentage}%)`;
+}
+
+async function findBuildJsonFiles(directory) {
```

Comment:
Could you put a comment to show how the directories look like here? it helps to understand how this function is traversing.